core/src/cluster_info.rs

```diff
@@ -51,6 +52,7 @@ use std::sync::atomic::{AtomicBool, Ordering};
 use std::sync::{Arc, RwLock};
 use std::thread::{sleep, Builder, JoinHandle};
 use std::time::{Duration, Instant};
+use tokio::net::TcpListener;
```

Comment:
Let's use `std::net::TcpListener` to prevent a tokio infection.  Within the ip echo server, we can use https://docs.rs/tokio/0.1.22/tokio/net/struct.TcpListener.html#method.from_std to go from a std TcpListener to a tokio one

Reply:
ah thank you